frontend/pages/SoftwarePage/components/AddSoftwareForm/helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { getPlatformDisplayName } from "utilities/file/fileUtils";
+
+import { IAddSoftwareFormData } from "./AddSoftwareForm";
+
+const FORM_VALIDATION_CONFIG = {
+  software: {
+    isValid: (formData: IAddSoftwareFormData) => formData.software !== null,
+  },
+  preInstallCondition: {
```

Comment:
Is this a TODO

Reply:
yep. i need some work from another task will integrate with that later.